tests/Microsoft.Identity.Web.Test/ClientInfoTests.cs

```diff
@@ -10,8 +10,8 @@ namespace Microsoft.Identity.Web.Test
 {
     public class ClientInfoTests
     {
-        private const string Uid = TestConstants.Uid;
-        private const string Utid = TestConstants.Utid;
+        private const string Uid = "Uid";
+        private const string Utid = "Utid";
```

Comment:
These two can still use TestConstants. The tests don't depend on the actual value.